Gemfile

```diff
@@ -55,6 +55,7 @@ group :development, :test do
 
   # Utilities
   gem 'figaro' # Simple Rails app configuration
+  gem 'mailgun-ruby' # Contains methods for easily interacting with the Mailgun API
```

Comment:
This should not be in the dev and test environments group only 🙈 This gem would not be installed in the prod environment. It should be moved in the block for `backend` (starting at line 4)